src/plugins/objects/mill/flour-bin-plugin.ts

```diff
@@ -1,25 +1,18 @@
-import { World } from '@server/world/world';
 import { ActionType, RunePlugin } from '@server/plugins/plugin';
 import { itemIds } from '@server/world/config/item-ids';
-import { itemOnObjectAction } from '@server/world/actor/player/action/item-on-object-action';
 import { objectAction } from '@server/world/actor/player/action/object-action';
-import { gameCache } from '@server/game-server';
-import { animationIds } from '@server/world/config/animation-ids';
 import { soundIds } from '@server/world/config/sound-ids';
-import { dialogueAction, DialogueEmote } from '@server/world/actor/player/action/dialogue-action';
-import { npcIds } from '@server/world/config/npc-ids';
+import { itemOnObjectAction } from '@server/world/actor/player/action/item-on-object-action';
 
 
-export const action: objectAction = (details) => {
+export const action: objectAction | itemOnObjectAction = (details) => {
```

Comment:
Change this to just `: itemOnObjectAction` or `: objectAction`. A single action can not have two types, you need to define two actions if they're of a different type.